testsuite/tests/letrec/underscore_binding.ml

```diff
@@ -0,0 +1,13 @@
+let p = print_endline
+
+(* Test "_" bindings in `let rec` constructs. All bindings produce the same
+ * effect to avoid testing evaluation order. *)
+let test =
+  let rec _ = p "1"; p "2"
+      and x = p "1\n2"; 3
```

Comment:
It'd be good to have the bindings used recursively in the test -- i.e. use `x` in the rhs of `y` and vice versa, and also use them both in the rhs of the `_` bindings.